brickschema/brickify/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+The subpackage `brickify` provides a set of handlers and a CLI tool
+for converting tabular or graph-based building data to Brick models.
+"""
+
+__version__ = "0.5.0"
```

Comment:
Does it make sense to keep the old `__version__` here? Might be easier to just use the `brickschema` version, but I'm open to ideas